src/pybind/mgr/dashboard_v2/README.rst

```diff
@@ -152,3 +152,55 @@ Example::
 
 Now only authenticated users will be able to "ping" your controller.
 
+
+How to access the manager module instance from a controller?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each controller class annoted with the ``ApiController`` decorator is injected
+with a class variable that points to the manager module global instance. That
+variable can be accessed using the ``mgr()`` class method.
+
+Example::
+
+  import cherrypy
+  from ..tools import ApiController, RESTController
```

Comment:
Is `..tools` correct? This looks strange to me.

Reply:
yes, it's python3 replative imports